pkg/util/csi_proxy_windows.go

```diff
@@ -35,27 +35,25 @@ type CSIProxy interface {
 	// GetVolumeStats gets the volume stats of a volume identified by `volumeId`
 	GetVolumeStats(volumeId string) (totalBytes int64, usedBytes int64, err error)
 
-	// FormatVolume formats a volume identified by `volumeId`
-	FormatVolume(volumeId string) (err error)
+	// RmdirContents removes the contents of a directory in the host filesystem.
+	RmdirContents(path string) (err error)
 
 	// IsSymlink checks if the given path is a symlink
 	IsSymlink(mountPath string) (isSymlink bool, err error)
+
+	// GetClosestVolumeIDFromTargetPath gets the closest volume id for a given target path
+	// by following symlinks and moving up in the filesystem, if after moving up in the filesystem
+	// we get to a DriveLetter then the volume corresponding to this drive letter is returned instead.
+	GetClosestVolumeIDFromTargetPath(targetPath string) (volumeID string, err error)
 }
 
 // NewCSIProxy returns an instance of the CSIProxy client compatible with either v1 or v1beta
 func NewCSIProxy() (CSIProxy, error) {
-	csiProxyV1, err := NewCSIProxyV1()
+	csiProxyV1, err := NewCSIProxyV2()
```

Comment:
changed the name to csiProxyV2 or csiProxyV2Alpha?

Reply:
thanks, fixed